lib/src/main/java/org/exoplatform/social/addons/updater/RDBMSMigrationManager.java

```diff
@@ -140,6 +156,10 @@ public void run() {
                       && !MigrationContext.isIdentityDone()) {
                 getIdentityMigrationService().start();
                 updateSettingValue(MigrationContext.SOC_RDBMS_IDENTITY_MIGRATION_KEY, MigrationContext.isIdentityDone());
+
+                if (useMigrationIdentityStorage && MigrationContext.isIdentityDone()) {
+                  identityManager.setIdentityStorage(identityStorage);
```

Comment:
I don't put in a finally block because we want to set the identity storage to the original only when the we migrate successfully for all identities. If any of them failed we will keep this service to continue migrate the remaining identities when they are needed.